src/fairness/weighted_tree/test/load_accounting_db.cpp

```diff
@@ -323,6 +324,7 @@ std::shared_ptr<weighted_tree_node_t> load_accounting_db (
     rc = sqlite3_open_v2 (path.c_str (), &DB, SQLITE_OPEN_READWRITE, NULL);
     if (rc != SQLITE_OK) {
         std::cerr << "error opening DB: " << sqlite3_errmsg (DB) << std::endl;
+        errno = EIO;
```

Comment:
Sorry I didn't leave the same comment for other places where sqlite functions are used and failed. You may want to add errno for those calls as well.

Reply:
Ah, sorry about that. 

I am looking at the different error codes available to return; I'm not sure `EOI` would be a good errno to use for the other `sqlite3_*` function calls. Does `EPROTO` make more sense to use in those other spots of code?

Reply:
Or EINVAL where it makes better sense...